src/jAlert.js

```diff
@@ -821,6 +830,55 @@
         }
     };
 
+    var toggleFocusTrap = function() {
+      let numVisiblejAlerts = $('.jAlert:visible').length;
+      let lastVisibleAlert = $('.jAlert:visible:last');
```

Comment:
I believe this can be replaced with:

let curAlert = $.jAlert('current');

The last alert is assumed to be the current so the logic is the same.